app/views/comments/_comment.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="comment">
+  <p><%= comment.body %></p>
+  <% comment.replies.each do |reply| %>
+    <%= render 'comments/comment', comment: reply %>
+  <% end %>
```

Comment:
I think that this render does not make any sense to render over the same partial.

Reply:
You're right. The main idea was to generate the form to respond to a parent comment, but in the end I ended up discarding it, and I forgot to remove it when creating the commit. Thx for the observation.